### ISSUE TITLE
Add docs for Data Enrichment

### ISSUE THREAD

**Issue by josephwegner** [keenlabs/KeenClient-Python#107]
Data Enrichment is a super useful feature when sending events to Keen IO. You can see more about it here: https://keen.io/docs/streams/data-enrichment-overview/. It would be super helpful in the **Send Events to Keen IO** section. 

In the past, I started docs for it, but I never finished and got a pull request merged, so here are some docs I wrote that you can choose to work off of: 

> #### Data enrichment
> 
> Keen IO can enrich event data by parsing or joining it with other data sets. This is done through the concept of “add-ons”.
> 
> To activate add-ons, you simply add some new properties within the “keen” namespace in your events. Detailed documentation for the configuration of our add-ons is available [here](https://keen.io/docs/api/?shell#data-enrichment).
> 
> Here is an example of using the [Datetime parser](https://keen.io/docs/api/?shell#datetime-parser). Let's assume you want to do a deeper analysis on the "purchases" event by day of the week (Monday, Tuesday, Wednesday, etc.) and other interesting Datetime components. 
> 
> ```python
>     keen.add_event("purchases", {
>       "keen": {
>         "addons": [
>           {
>             "name": "keen:date_time_parser",
>             "input": {
>               "date_time" : "keen.timestamp"
>             },
>             "output": "timestamp_info"
>           }
>         ]
>       },
>       "price": 500
>     })
> ```
> 
> Other data enrichment add-ons are located in the [API reference docs](https://keen.io/docs/api/?shell#data-enrichment).

(@tbarn: copy and pasted from my comment so it shows up in the description)

**Comment by josephwegner** [comment]
#105 will close this

**Comment by tbarn** [comment]
Removed a comment about a PR that would fix this but didn't get merged to avoid confusion :)

**Comment by tbarn** [comment]
Data Enrichment is a super useful feature when sending events to Keen IO. You can see more about it here: https://keen.io/docs/streams/data-enrichment-overview/

In the past, I started docs for it, but I never finished and got a pull request merged, so here are some docs I wrote that you can choose to work off of: 

#### Data enrichment

Keen IO can enrich event data by parsing or joining it with other data sets. This is done through the concept of “add-ons”.

To activate add-ons, you simply add some new properties within the “keen” namespace in your events. Detailed documentation for the configuration of our add-ons is available [here](https://keen.io/docs/api/?shell#data-enrichment).

Here is an example of using the [Datetime parser](https://keen.io/docs/api/?shell#datetime-parser). Let's assume you want to do a deeper analysis on the "purchases" event by day of the week (Monday, Tuesday, Wednesday, etc.) and other interesting Datetime components. 

```python
    keen.add_event("purchases", {
      "keen": {
        "addons": [
          {
            "name": "keen:date_time_parser",
            "input": {
              "date_time" : "keen.timestamp"
            },
            "output": "timestamp_info"
          }
        ]
      },
      "price": 500
    })
```

Other data enrichment add-ons are located in the [API reference docs](https://keen.io/docs/api/?shell#data-enrichment).

**Comment by mamat-rahmat** [comment]
I'd like to work on this. Just to make sure, do you mean [this part](https://github.com/keenlabs/KeenClient-Python#send-events-to-keen-io)?

**Comment by tbarn** [comment]
Yes, at the bottom here: 

<img width="482" alt="screen shot 2017-10-05 at 3 00 08 pm" src="https://user-images.githubusercontent.com/2008034/31250227-cb3bd358-a9df-11e7-89f9-77153e1a90f0.png">

Yup, in its own section right before it gets into analysis.

**Comment by mamat-rahmat** [comment]
I think for code example, i would use [URL Parser](https://keen.io/docs/streams/data-enrichment-overview/#addon-url-parser) as it simple enough to explain the idea. What do you think?

**Comment by tbarn** [comment]
Sure. It might be helpful to use that one and another one, especially since the Datetime Parser docs have been already partially written.

**Comment by tbarn** [comment]
Closed in #136
